frontend/src/service-worker.js

```diff
@@ -1,15 +1,72 @@
-import {serviceWorkerFetchListener} from "sync-message";
+/* eslint-disable */
+// Otherwise webpack can fail silently
+// https://github.com/facebook/create-react-app/issues/8014
 
-console.log(self.__WB_MANIFEST);
+// This service worker can be customized!
+// See https://developers.google.com/web/tools/workbox/modules
+// for the list of available Workbox modules, or add any other
+// code you'd like.
+// You can also remove this file if you'd prefer not to use a
+// service worker, and the Workbox build step will be skipped.
 
-const fetchListener = serviceWorkerFetchListener();
+import { clientsClaim } from 'workbox-core';
+import { ExpirationPlugin } from 'workbox-expiration';
+import { precacheAndRoute, createHandlerBoundToURL } from 'workbox-precaching';
+import { registerRoute } from 'workbox-routing';
+import { StaleWhileRevalidate } from 'workbox-strategies';
 
-addEventListener('fetch', fetchListener);
```

Comment:
Keep this, sync-message is still important

Reply:
Good catch — I replaced `service-worker.js` wholesale with the one from `cra-template-pwa`. Will put that back.